proctor-common/src/test/java/com/indeed/proctor/common/TestTestRangeSelector.java

```diff
@@ -128,19 +142,25 @@ public void testFindMatchingRule_UnitlessAllocation_NotUnitless() {
     }
 
     private static TestDefinition.Builder stubTestDefinition(final List<String> rules) {
-        return stubTestDefinition(rules, false);
+        return stubTestDefinition(rules, false, null);
     }
 
     private static TestDefinition.Builder stubTestDefinition(
             final List<String> rules, final boolean unitless) {
+        return stubTestDefinition(rules, false, null);
+    }
+
+    private static TestDefinition.Builder stubTestDefinition(
+            final List<String> rules, final boolean unitless, @Nullable final String rule) {
         return TestDefinition.builder()
+                .setRule(rule)
                 .setTestType(TestType.ANONYMOUS_USER)
                 .setSalt("")
                 .setEnableUnitlessAllocations(unitless)
                 .addBuckets(new TestBucket("active", 1, ""))
                 .setAllocations(
                         rules.stream()
-                                .map(rule -> new Allocation(rule, singletonList(new Range(1, 1.0))))
+                                .map(r -> new Allocation(r, singletonList(new Range(1, 1.0))))
```

Comment:
Is this mostly a style preference change, or a lint fix type change?

Reply:
I can improve the name. I add in passing a test level rule in the parameters as rule. I can make that `testRule` and remake this `rule -> `.

Reply:
I can actually clean this up. No longer need the helper after I refactored the test case.